tools/apps/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "quiche_apps"
+version = "0.1.0"
+authors = ["Lucas Pardue <lucaspardue.24.7@gmail.com>"]
+edition = "2018"
+publish = false
+
+[dependencies]
+docopt = "1"
+env_logger = "0.6"
+http = "0.2"
```

Comment:
Is this used anywhere?

Also, `ring` and `serde*`

Reply:
ring is used by quiche-server, the others are bogus